product_mandatory_products/static/src/js/website_sale_options.js

```diff
@@ -0,0 +1,106 @@
+odoo.define("product_mandatory_products.website_sale_options", function (require) {
+  "use strict";
+
+  var ajax = require("web.ajax");
+  var rpc = require("web.rpc");
+  var publicWidget = require("web.public.widget");
+  var OptionalProductsModal = require("sale_product_configurator.OptionalProductsModal");
+  require("website_sale_options.website_sale");
+
+  publicWidget.registry.WebsiteSale.include({
+    //    _onProductReady: function () {
+    //        debugger;
+    //        var res =  this._super.apply(this, arguments);
+    //        return res
+    //    },
```

Comment:
Do you think there might be a future use for this commented code or can it be removed?